package/MDAnalysis/core/groups.py

```diff
@@ -601,7 +601,7 @@ def dimensions(self):
     def dimensions(self, dimensions):
         self.universe.trajectory.ts.dimensions = dimensions
 
-    def center(self, weights, pbc=None):
+    def center(self, weights, pbc=None, compound='group'):
```

Comment:
Appending the new `compound` kwarg as the last parameter seems good to me. This way, code implicitly using the `pbc` kwarg won't be broken.

Reply:
yeah good idea.

Reply:
Just to clarify: With "implicitly using the `pbc` kwarg" I meant calling `center_of_mass(True)` instead of `center_of_mass(pbc=True)`.